src/build.rs

```diff
@@ -86,7 +87,63 @@ fn patch_lib_kind_in_target(ws: &mut Workspace, libkinds: &[&str]) -> anyhow::Re
     Ok(())
 }
 
-/// Build import library for Windows
+/// Build def file for windows-msvc
+fn build_def_file(
+    ws: &Workspace,
+    name: &str,
+    target: &target::Target,
+    targetdir: &PathBuf,
+) -> anyhow::Result<()> {
+    let os = &target.os;
+    let env = &target.env;
+
+    if os == "windows" && env == "msvc" {
+        ws.config()
+            .shell()
+            .status("Building", ".def file using dumpbin")?;
+
+        let txt_path = targetdir.join(format!("{}.txt", name));
+        let mut dumpbin = std::process::Command::new("dumpbin");
+        dumpbin
+            .arg("/EXPORTS")
+            .arg(targetdir.join(format!("{}.dll", name)));
+        dumpbin.arg(format!("/OUT:{}", txt_path.to_str().unwrap()));
+
+        let out = dumpbin.output()?;
+        if out.status.success() {
+            let txt_file = File::open(txt_path)?;
+            let buf_reader = BufReader::new(txt_file);
+            let mut def_file = File::create(targetdir.join(format!("{}.def", name)))?;
+            writeln!(def_file, "{}", "EXPORTS".to_string())?;
+
+            // Skip 16 lines and, for each line, delete all the characters
```

Comment:
Is this true for all supported versions of `dumpbin`? Same for below. It seems possibly a little brittle to hardcode numbers of lines or spacs.

Reply:
I've modified that code starting from the following loop:

`for /f "skip=19 tokens=4" %A in (file.txt) do echo %A >> file.def`

This loop skips 19 lines because `dumpbin` adds 3 lines of copyright information that are printed on the shell when we use the `/OUT:file.txt' option. Searching through the net I've always found that loop above, so we could suppose that is true for all supported versions of `dumpbin`.